### PR TITLE
Yet more bone simulator fields

### PR DESCRIPTION
**Pull request by redstrate**
A few more fields I found that are somewhat useful, I guess

### CODE REVIEW

File: FFXIVClientStructs/FFXIV/Client/Graphics/Physics/BoneSimulator.cs [hunk 1]
```diff
@@ -8,7 +8,21 @@ namespace FFXIVClientStructs.FFXIV.Client.Graphics.Physics;
...
+        HairA = 3, // Usually the hair's bangs
+        HairB = 4, // Typically the back of the hair
+        HairC = 5, // The sides of the hair
+        HairD = 6, // Extraneous hair bits
+        Chest = 7,
+        Earrings = 8,
+        Ears = 18,
+    }
+
+    [FieldOffset(0x10)] public PhysicsGroup Group;
```
Comment (redstrate) [inline FFXIVClientStructs/FFXIV/Client/Graphics/Physics/BoneSimulator.cs:25]: Don't know the rules against API changes, is it OK to break? (I'm not sure if there's even any consumers)

File: FFXIVClientStructs/FFXIV/Client/Graphics/Physics/BonePhysicsModule.cs [hunk 2]
```diff
@@ -18,6 +18,7 @@ public struct BoneSimulators {
 public unsafe struct BonePhysicsModule {
     [FieldOffset(0x00)] public void* vtbl;
     [FieldOffset(0x10)] public Matrix4x4 SkeletonWorldMatrix;
+    [FieldOffset(0x40)] public Vector3 CharacterPosition;
```
Comment (wolfcomp) [inline FFXIVClientStructs/FFXIV/Client/Graphics/Physics/BonePhysicsModule.cs:21]: This is part of the `Matrix4x4` if you want to have just the vector3 component out, make a getter function for this instead.

**Review by wolfcomp — CHANGES_REQUESTED** [review]

File: FFXIVClientStructs/FFXIV/Client/Graphics/Physics/BoneSimulator.cs (hunk 1 above)
Comment (wolfcomp) [reply FFXIVClientStructs/FFXIV/Client/Graphics/Physics/BoneSimulator.cs:25]: ABI changes will be auto marked as breaking change due to how C# works, even if there are no consumers for this. This is due to the check system is just comparing it between the old and new CS versions.

File: FFXIVClientStructs/FFXIV/Client/Graphics/Physics/BonePhysicsModule.cs (hunk 2 above)
Comment (redstrate) [reply FFXIVClientStructs/FFXIV/Client/Graphics/Physics/BonePhysicsModule.cs]: Oops yes, I miscalculated some of the sizes. I reverted this bit

File: FFXIVClientStructs/FFXIV/Client/Graphics/Physics/BoneSimulator.cs (hunk 1 above)
Comment (redstrate) [reply FFXIVClientStructs/FFXIV/Client/Graphics/Physics/BoneSimulator.cs:25]: Ah ok, cool!

**Pull request merged**